shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -689,8 +690,21 @@ - (void)viewDidLayoutSubviews {
 
   // This must run after updateViewportMetrics so that the surface creation tasks are queued after
   // the viewport metrics update tasks.
-  if (firstViewBoundsUpdate)
+  if (firstViewBoundsUpdate) {
     [self surfaceUpdated:YES];
+
+    flutter::Shell& shell = [_engine.get() shell];
+    fml::TimeDelta waitTime =
+#if NDEBUG
+        fml::TimeDelta::FromMilliseconds(100);
+#else
+        fml::TimeDelta::FromMilliseconds(200);
+#endif
+    if (shell.WaitForFrameRender(waitTime)) {
```

Comment:
@liyuqian I was just looking at the Android API and noticed they have the first frame callback and we never implemented it for iOS.  Potentially that is something I could hook onto instead of the flush I'm currently doing.  I'll look into that next week.